odoo/addons/base/models/ir_attachment.py

```diff
@@ -715,3 +715,15 @@ def action_get(self):
     def _get_serve_attachment(self, url, extra_domain=None, order=None):
         domain = [('type', '=', 'binary'), ('url', '=', url)] + (extra_domain or [])
         return self.search(domain, order=order, limit=1)
+
+    def _compute_image_url(self, b64image):
```

Comment:
@ryv-odoo I was quite confused by that request from @tde-banana-odoo so I just added it in a temporary commit to see what he meant (see here: https://github.com/odoo/odoo/pull/119325#discussion_r1200270471).

Reply:
Obviously if you don't use it anymore because it is easier without it, no need to define it :) last version of code seems well contained so probably unnecessary to define it.